src/components/sgc-menu-item/sgc-menu-item.css

```
@@ -0,0 +1,55 @@
+:host {
```

Add the base import here, mainly to get `box-sizing: border-box` for the item's Shadow DOM.
```suggestion
@import "../../global/base.css";

:host {
```